src/test/java/com/jcabi/w3c/DefaultHtmlValidatorTest.java

```diff
@@ -51,6 +51,14 @@
  *
  * @author Yegor Bugayenko (yegor@tpc2.com)
  * @version $Id$
+ * @todo #1:30min Invalid html response test file need to be fixed.
+ * File invalid-html-response.xml-without-schema contains valid content
```

Comment:
@prondzyn wrong indentation on puzzle here. Text on subsequent lines should be aligned to `t' instead of `@`, like so:

```java
* @todo #1:30min Invalid html response test file need to be fixed.
*  File invalid-html-response.xml-without-schema contains valid content
```